CMakeLists.txt

```diff
@@ -137,6 +137,7 @@ target_link_libraries(monitor
 
 target_include_directories(monitor PRIVATE
   ${QWT_INCLUDE_DIR}
+  ${PROJECT_SOURCE_DIR}/thirdparty/OpenDDW/src
```

Comment:
Linking to OpenDDW should take care of this. What happens without it?

Reply:
It doesn't build without it.  The directories don't line up right due to the directory layout.

Reply:
In CMake if you have to use `target_include_directories` on a library you're already using `target_link_libraries` on means something isn't right in that library's CMake files. What is the exact error?

Reply:
The cmake is looking in the install directory which doesn't have the include files until the install phase.  That works fine when you are using an installed package.  But we need to include the source directory manually for using it as a submodule.

Reply:
It's part of the monitor build, so there shouldn't be any reason for it to be looking at the installed location. I built this locally to see what it's doing and I see what it's doing now after looking at the `-I` options it passed to the compiler. It used to be including `src` before https://github.com/OpenDDS/OpenDDW/pull/13 and now it's just `${CMAKE_CURRENT_SOURCE_DIR}`.

Changing that to `${CMAKE_CURRENT_SOURCE_DIR}/src` in OpenDDW allows the monitor to build without this added line. I've created a OpenDDW PR that does this: https://github.com/OpenDDS/OpenDDW/pull/15.

Reply:
OK but the source directory isn't necessary to build OpenDDW or to use it from an installed location.  Its only a concern if you are using it as a submodule.

Reply:
The include would be required for every target in the same build that wants to use OpenDDW. If a test was added to OpenDDW it would have the same problem. It's better to fix it once than to require a manual include for every target.

Reply:
I'll remove the directory and bump the submodule once the other pull request is merged.

Reply:
Any chance this can be moved along.  These changes are about as trivial as they come. It would be nice to have a version that builds again.